index.html

```diff
@@ -2499,7 +2501,7 @@ <h3 id="7_C_1">7.C.1 Overview</h3>
                         <strong>package_type</strong><br>
                         Data type: <a href="#package_type">package_type</a><br>
                         Required: yes<br>
-                        The package type. For international shipment only package type 1 is allowed.
+                        The package type. For international shipment only package type 1 (package) is allowed.

```

Comment:
@reindert-vetter to indicate that package type 1 = package!
Otherwise I don't understand your question.